### PR TITLE
Add missing links to logos and remove ripple effect

### PR DESCRIPTION
**Pull request by Pallavi-PH**
1. This PR adds missing links to logos
2. Removes ripple effect from next & previous buttons in blog pages

### CODE REVIEW

File: website/src/components/Sponsor/index.tsx [hunk 1]
```diff
@@ -23,19 +23,23 @@ const Sponsor: React.FC = () => {
           <Grid container>
             <Grid item xs={12} md={6} sm={6}>
               <Typography className={classes.paragraph}>{t("sponsors.mayadata")}</Typography>
-              <img
-                src="../images/logos/mayadata_logo.svg"
-                alt={t("generic.mayadata")}
-                className={classes.sponsorCompany}
-              />
+              <a href="https://mayadata.io/" rel="noreferrer" target="_blank">
```
Comment (IsAmrish) [inline website/src/components/Sponsor/index.tsx:26]: Please take `https://mayadata.io/` and `https://www.cncf.io/` from constants file

Comment (IsAmrish) [reply website/src/components/Sponsor/index.tsx:26]: <Link /> from `material-ui` can be used for external links also

File: website/src/pages/CommercialSupport/index.tsx [hunk 2]
```diff
@@ -113,12 +113,16 @@ const Support: React.FC = () => {
                   <Card className={classes.cardProps} key={elm.id}>
                     <CardContent>
                       <div className={classes.cardImageWrapper}>
-                        <img
-                          loading="lazy"
-                          src={elm.image}
-                          alt={elm.name}
-                          
-                        />
+                        <a href={elm.website}
```
Comment (IsAmrish) [inline website/src/pages/CommercialSupport/index.tsx:116]: please use `Link` from material ui

File: website/src/components/Sponsor/index.tsx (hunk 1 above)
Comment (satyapriyamishra222) [reply website/src/components/Sponsor/index.tsx:26]: Let's use the Link element

File: website/src/pages/CommercialSupport/index.tsx [hunk 3]
```diff
@@ -113,12 +114,16 @@ const Support: React.FC = () => {
...
-                          
-                        />
+                        <Link href={elm.website}
+                        rel="noopener noreferrer"
+                        target="_blank">
+                          <img
+                            loading="lazy"
+                            src={elm.image}
+                            alt={elm.name}
+
```
Comment (rakeshPRaghu) [inline website/src/pages/CommercialSupport/index.tsx:124]: Remove the empty line

Comment (Pallavi-PH) [reply website/src/pages/CommercialSupport/index.tsx]: Removed

**Review by rakeshPRaghu — APPROVED** [review]

File: website/src/components/Sponsor/index.tsx (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/components/Sponsor/index.tsx]: Done

**Review by IsAmrish — APPROVED** [review]
Lgtm

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**